main.js

```diff
@@ -1,3 +1,66 @@
 import './style.css';
 
-console.log('Hello world');
+console.log(123123123);
+
+const btn1 = document.getElementById("btn1");
+const tt = document.getElementById("tt");
```

Comment:
Using a one element for all tooltips is a good decision for this task, but it is not good way for making tooltip in common. It is better to create tooltip on fly if it is really used.
I guess if you will click on second button and then hover the first - first button will steal the tooltip and it is not good behaviour )